news/2018-01-20-beatmap-spotlights-december-2017.md

```diff
@@ -0,0 +1,152 @@
++ You must map **at least three** (3) difficulties in an appropriate spread as per the Ranking Criteria, either ENH (easy, normal and hard) or NHI (normal, hard, insane).
++ Use SB01 as a tag for easier find.
++ Do not submit mapsets directly to us, instead ask the Beatmap Nominators for help!
++ Your mapset must be ranked before the 15th of March!
+
+## What are the Prizes?
+
++ **1st place:** 3 months of osu!supporter and a shiny new badge, "Bounty Hunter: Thaehan"
++ **2nd place:** 2 months of osu!supporter
++ **3rd place:** 1 month of osu!supporter
++ The prizes are awarded per game mode
+
+## What are the Prizes?
+
++ **1st place:** 3 months of osu!supporter and 15 Bounty point
++ **2nd place:** 2 months of osu!supporter and 12 Bounty point
++ **3rd place:** 1 month of osu!supporter and 9 Bounty point
++ The points will go down in a respective order
++ At the end of the year these points will decide who will go home with the badge and title "Bounty Hunter 2018: osu!/taiko/catch/mania"
++ The prizes are awarded per game mode
++ All players will be able to earn a shiny new medal "Bounty Hunter: Thaehan".
+
+# December 2017 Spotlights
+
+## osu!
+
+[![](/wiki/shared/news/2018-01-20-beatmap-spotlights-december-2017/impulse.jpg)](https://osu.ppy.sh/beatmapsets/705788)
+
+As one of the most well known mappers in the community, [handsome](https://osu.ppy.sh/users/2123087) continues to impress us with his latest ranked map - Impulse after RELOADED! Featuring 3 specific types of sliders coupled with delightful usage of patterns and small circle size, this map gives a uniquely aim-heavy challenge that fits the song at its finest! Be sure to check out the alluring storyboard made by Darky1 as well.
+
+[![](/wiki/shared/news/2018-01-20-beatmap-spotlights-december-2017/jin-li-chao.jpg)](https://osu.ppy.sh/beatmapsets/684393)
+
+[AIR](https://osu.ppy.sh/users/2070688)'s breathtaking virtuosity with sliders are showcased in his latest set of Jin Li Chao. Movements flow effortlessly while still captivating the player throughout an equally elegant song. It is uncommon to see such a difficult map of a rather simple song, but AIR has tackled it brilliantly. Definitely give this one a try if you haven't done so!
+
+[![](/wiki/shared/news/2018-01-20-beatmap-spotlights-december-2017/yoiyoi-kokon.jpg)](https://osu.ppy.sh/beatmapsets/545737)
+
+On the song YoiYoi Kokon by Japanese singer REOL, [DreaM117er](https://osu.ppy.sh/users/1945090) brings us a mapset full of great difficulties. The highest difficulty made by the infamous [kiddly](https://osu.ppy.sh/users/74937) is packed with its own twists on popular patterns like sliderstreams, while the mapset host's own Expert difficulty manages to impress with its clean patterning and great flow. Add a stunning storyboard by [StarrodKirby86](https://osu.ppy.sh/users/410), combined with a beautiful skin by [Chaoslitz](https://osu.ppy.sh/users/3621552), and you have a mapset that’s not to be missed out this month!
```

Comment:
First part of this paragraph is a bit 🤔, would delete and do
"DreaM117er's YoiYoi Kokon brings us" works fine